bin/daily_total.py

```diff
@@ -63,11 +66,22 @@ def main():
             LOGGER.warning(msg)
             return
     # Format the data
-    data = "New issues on {yesterday}: {issue_count}".format(
-        yesterday=yesterday.isoformat(), issue_count=issue_count
-    )
-    # Log the data on the console
-    print(data)
+    yesterday = yesterday.isoformat()
```

Comment:
We can probably create a `iso_yesterday` at the top, just after
`yesterday = today - datetime.timedelta(days=1)`

and then reuse this anywhere the code uses `yesterday.isoformat()`, There are already 3 instances.